subprojects/internal-architecture-testing/src/main/java/org/gradle/internal/architecture/PackageCycleTest.java

```diff
@@ -63,9 +68,7 @@ private static boolean isClassNamePattern(String pattern) {
     }
 
     private static Set<String> ignoredClassesForCycles() {
-        String patterns = System.getProperty("package.cycle.exclude.patterns");
-        return Arrays.stream(patterns.split(" "))
```

Comment:
❔  Interesting, I suppose this never worked. Do we have any subprojects that add multiple classes?

Reply:
> ❔ Interesting, I suppose this never worked. Do we have any subprojects that add multiple classes?

We have one that combines class and package exclude, but it worked because class exclude is covered by the package one. I added mine to another project and it didn't work.